### ISSUE TITLE
Add accessibility updates to language superscripts

### ISSUE THREAD

**Issue by kamerynB** [project-lux/lux-frontend#292]
**Problem Description:**
Language superscripts and data points with specific languages are missing accessibility features.

**Expected Behavior/Solution:**
Language superscripts and langauge specific data points should have specific accessibility tags added to the HTML.

**Requirements:**

- [ ] Add the `lang` attribute to the data points that are implementing different languages (useful for screen readers)
- [x] Make the language superscript color darker for contrast improvments

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Kam
- [x] ~Committee discussions - Sarah~
- [x] ~Feasibility/Team discussion - Sarah~
- [x] ~Backend requirements - TBD~
- [x] Frontend requirements - Kam
- [ ] Are new regression tests required for QA - Amy
- [x] ~Questions~

**UAT/LUX Examples:**

~- https://lux-front-tst.collections.yale.edu/view/person/34f4eec7-7a03-49c8-b1be-976c2f6ba6ba~
- ID for Andy Warhol has changed: https://lux-front-tst.collections.yale.edu/view/person/66049111-383e-4526-9632-2e9b6b6302dd

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
N/A

**Related links:**
N/A

**Wireframe/Mockup:**
Current: 
![image](https://github.com/user-attachments/assets/dce233eb-a504-463b-9cd4-439f70e3ffdb)

Expected: 
color of the links will be a slightly darker shade of what it is currently and have updated accessibility language superscripts.

**Comment by roamye** [comment]
@kamerynB - will FE contain to remain the same? I assume so, but wanted to verify.

**Comment by kamerynB** [comment]
It will change slightly. @roamye The color of the links will be a slightly darker shade of what it is currently.

**Comment by kamerynB** [comment]
Update 3/26/2025:
The `lang` attribute requires [specific language codes](https://www.w3schools.com/tags/ref_language_codes.asp) that we don't have. I reached out to Nick and Kara to determine if there is another approach to take. I am awaiting a response.

**Comment by roamye** [comment]
4/10 Approved by UAT

**Comment by roamye** [comment]
Looks good in PRD. Closing.
